public/app/features/admin/AdminListOrgsPage.tsx

```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
       <Page.Contents>
         <>
           <div className="page-action-bar">
+            <InfoBox branded>
+              <VerticalGroup spacing="xs">
+                <Badge text="Info" color="blue" icon="info-circle" />
+                <p>
+                  Organizations is a feature that only 1% of Grafana users are utilizing, and we feel that most of those
+                  users would have a better experience using Teams instead. As such, we are considering de-emphasizing
+                  and eventually deprecating Organizations in future Grafana releases. If you would like to provide
+                  feedback or describe your use case, please do so{' '}
+                  <a className="external-link" href="https://github.com/grafana/grafana/issues/xxx">
```

Comment:
Could you please create this issue now? Would be nice to have a real issue to link to so we don't forget to update it :)

Reply:
The issue can be named placeholder for now since it doesn't affect the link.

Reply:
added issue link